tools/cvm-image-rewriter/plugins/03-netplan/README.md

```diff
@@ -0,0 +1,3 @@
+# Netplan
+
+This plugin is used to add network configuration file netplan.yaml in the `/etc/netplan/` directory, which will configure the network for the guest image. Users can customize netplan.yaml by updating the contents of `files/etc/netplan/netplan.yaml`
```

Comment:
I would change this to 

This plugin is used to add network a network configuration file `netplan.yaml` in the ...   I would also put code quotes around the other netplan.yaml